src/Sulu/Component/HttpCache/EventSubscriber/UpdateResponseSubscriber.php

```diff
@@ -1,68 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Component\HttpCache\EventSubscriber;
-
-use Sulu\Component\HttpCache\HandlerUpdateResponseInterface;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
-use Symfony\Component\HttpKernel\KernelEvents;
-
-/**
- * Update the website response with cache headers.
- */
-class UpdateResponseSubscriber implements EventSubscriberInterface
```

Comment:
Are you sure we can delete this file?

Reply:
this service is not registered and the constructor param `HandlerUpdateResponseInterface` not longer exists /cc @trickreich